scripts/run_notebooks_timeouts.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
Nitpick: I'd advocate for `pathlib.Path` as a modern high-level replacement for `os.path`

Reply:
done